multiversx_sdk_cli/contracts.py

```diff
@@ -36,6 +36,8 @@
 FALSE_STR_LOWER = "false"
 TRUE_STR_LOWER = "true"
 STR_PREFIX = "str:"
+ADDRESS_PREFIX = "addr:"
+MAINCHAIN_ADDRESS_HRP = "erd"
```

Comment:
Can we remove this constant, and directly use the field from `LibraryConfig` below?

Reply:
hmm, I am not sure about this. What if somebody changes the `LibraryConfig`?

Reply:
Generally speaking, since `mxpy` is the _end application_ (not a library), only us can be that somebody.

If someone changes the HRP in mxpy's config (legacy) or via the CLI (I think not available on all commands for now), she will also get the benefice of this legacy flow (if you would remove the constant and directly use the field from library config).

Having "erd" only for this legacy flow might be misleading & might lead to additional questions / support. Even if the flow is legacy.